### PR TITLE
Add security policy

### PR DESCRIPTION
**Pull request by jaimergp**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

cc @conda-forge/core

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      PR](https://docs.github.com/en/code-security/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability#creating-a-temporary-private-fork).
+    - **Coordinator**/**Developer**: If appliccable, request a
+      [CVE](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers)
+      from GitHub. The CVE Number will be private until the advisory is published.
+    - **Developer & Coordinator**: Decide on release and announcement dates and post them the draft
+      advisory.
+  - **Coordinator**: Post the release and announcement dates on the conda-forge core chat room and
+    mailing list.
+  - **Developer**: Merge the security fix PR
+  - **Developer**: Make a release to PyPI and/or npm with no announcement or change log
```
Comment (beckermr) [inline SECURITY.md:97]: This doesn't really apply or needs to be changed to be more general.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  - **Coordinator**: Publish the security advisory on the announcement date.  GitHub will post the
+    CVE to the MITRE database
+- **Coordinator**: Notify the **Reporter** of the releases
+- **Coordinator**: Close the issue in the tracking repository
+
```
Comment (beckermr) [inline SECURITY.md:102]: We should add a step where we possibly post on our blog, make an announcement, and/or post on element.

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+> - Also, vulnerabilities may involve multiple private security forks across different GitHub
+>   organizations.
+> - This may require additional manual steps to include those private forks.
+
+[coordinated-disclosure]: https://cheatsheetseries.owasp.org/cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.html#responsible-or-coordinated-disclosure
+
+<!--
+Heavily inspired by the excellent document used by the Jupyter community
+https://github.com/jupyter/security/blob/86ec517/docs/vulnerability-handling.md
+-->
```
Comment (beckermr) [inline SECURITY.md:116]: Let's make this text not a comment so people see it.

**Review by beckermr — COMMENTED** [review]
Looks great! Just a few comments.

File: SECURITY.md (hunk 1 above)
Comment (jaimergp) [reply SECURITY.md:97]: True! Changed it to "release and/or deploy"

File: SECURITY.md (hunk 2 above)
Comment (jaimergp) [reply SECURITY.md:102]: Added, thanks!

Comment (jaimergp) [reply SECURITY.md:107]: Done!

File: SECURITY.md (hunk 1 above)
Comment (jaimergp) [reply SECURITY.md]: True! Changed it to "release and/or deploy as needed"

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
Do we want to get comments from the rest of @conda-forge/core or are we ok to merge? I don't think this needs a vote, but others might feel differently.

**Comment by jaimergp** [comment]
Let's wait a bit, it won't hurt to have it open for 48h or so, I guess!

**Pull request merged**